src/openparse/tables/table_transformers/ml.py

```
@@ -27,62 +27,84 @@
+    TableHeaderCell,
+    TableRow,
 )
 
-t0 = time.time()
-device = config.get_device()
+# ================================
+# === GLOBAL SETUP & CONSTANTS ===
+# ================================
+t0: float = time.time()
+device: Literal['cuda'] | Literal['cpu'] = config.get_device()
+Size = Tuple[int, int]
```

Ruff (0.11.9)

Redefinition of unused `Size` from line 18

(F811)